linuxdeploy-plugin-gstreamer.sh

```diff
@@ -105,6 +106,8 @@ done
 
 for i in "$plugins_target_dir"/*; do
     [ -d "$i" ] && continue
+    [ ! -f "$i" ] && echo "File does not exist: $i" && continue
+    $(file "$i" | grep -v ELF --silent) && echo "Ignoring non ELF file: $i" && continue
```

Comment:
You need to escape the `$(...)` subshell:

```suggestion
    "$(file "$i" | grep -v ELF --silent)" && echo "Ignoring non ELF file: $i" && continue
```

[shellcheck](https://github.com/koalaman/shellcheck) is a useful companion for checking scripts.